scripts/environment.sh

```diff
@@ -24,10 +24,10 @@ function os_check() {
     export DISTRO="${ID}${VERSION_ID%.*}"
     export OS="${ID}"
     export OS_VERSION_ID=$VERSION_ID
-    export SUPPORTED_DISTROS=(centos8 centos9 rhel8 ubuntu20)
+    export SUPPORTED_DISTROS=(centos8 centos9 rhel8 ubuntu20 ubuntu22)
 
     if [[ ! "${SUPPORTED_DISTROS[*]}" =~ $DISTRO ]]; then
-        echo "Supported OS distros for the host are: CentOS Stream 8/9 or RHEL8/9 or Ubuntu20.04"
+        echo "Supported OS distros for the host are: CentOS Stream 8/9 or RHEL8/9 or Ubuntu20.04 or Ubuntu22.04"
```

Comment:
Agree. Lets first merge this to get the CI going and then we can remove it form here.

Reply:
but we still need to update the image url anyway ! but at least the test will not be broken because the new image is not supported but it continues working but downloading an old image for the nodes